src/ssh/test-sshbridge.c

```diff
@@ -1388,6 +1426,13 @@ main (int argc,
               setup, test_echo_and_close, teardown);
   g_test_add ("/ssh-bridge/ssh-config-home-invalid-user", TestCase, &fixture_home_ssh_config_invalid_user,
               setup, test_problem, teardown);
+  g_test_add ("/ssh-bridge/ssh-config-home-good-key", TestCase, &fixture_home_ssh_config_good_key,
+              setup, test_echo_and_close, teardown);
+  g_test_add ("/ssh-bridge/ssh-config-home-bad-key", TestCase, &fixture_home_ssh_config_bad_key,
+              setup, test_problem, teardown);
+  g_test_add ("/ssh-bridge/ssh-config-home-key-password-fallback", TestCase, &fixture_home_ssh_config_key_password_fallback,
+              setup, test_echo_and_close, teardown);
+
 
```

Comment:
Can you add a few cases for IdentityFile in the config and a separate identity file directly provided?  Just to ensure both are used, regardless of the order specified (or whatever the correct behaviour should be in this case).